CMakeLists.txt

```diff
@@ -40,6 +40,13 @@ include_directories(ext/qoi)
 
 include_directories(ext/cxxopts/include)
 
+# avif
+
+set(AVIF_CODEC_AOM LOCAL CACHE INTERNAL "" FORCE)
+set(AVIF_CODEC_DAV1D LOCAL CACHE INTERNAL "" FORCE)
+set(AVIF_LIBYUV LOCAL CACHE INTERNAL "" FORCE)
+add_subdirectory(ext/libavif)
```

Comment:
This is modeled after the jxl section below.

I am not a CMake expert. We need to pass these options (lines 45-47) to libavif's cmake configure command. I wonder if there is a simpler way to do this than these `set` commands.

Reply:
> I wonder if there is a simpler way to do this than these set commands.

I have not found one. Let me know if you do!